doc/Advanced-configuration.md

```diff
@@ -398,6 +398,61 @@ If they are not present - defaults are used (connection to `localhost:9042` with
  ]}.
 ```
 
+#### SSL connection setup
+
+In order to establish a secure connection to Cassandra you must make some changes in the MongooseIM and Cassandra configuration files.
+
+##### Create MongooseIM keyspace
```

Comment:
This section should go to https://github.com/esl/MongooseIM/blob/master/doc/advanced-configuration/database-backends-configuration.md